.github/workflows/update-main-source.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     - name: Update main-source branch
       run: |
         scripts/pull-ci-scripts
-        bin/partner-charts-ci auto
+        bin/partner-charts-ci auto --icons
```

Comment:
You will need to cut a new release of `partner-charts-ci`

Reply:
`scripts/pull-ci-scripts` will pull the latest one: https://github.com/rancher/partner-charts/blob/main-source/scripts/pull-ci-scripts
See line 5: `CI_BINARY_URL_BASE="https://github.com/rancher/partner-charts-ci/releases/latest/download"`

I just have to merge the `partner-charts-ci` PR first: https://github.com/rancher/partner-charts-ci/pull/3

Do you agree? @recena

Reply:
If it triggers a release process, it is okay.